src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/replication/ReplicationFailedHandler.java

```diff
@@ -40,6 +40,7 @@
 @Extension
 public class ReplicationFailedHandler extends RunListener<AbstractBuild<?, ?>> {
 
+    // TODO: scooby doo'd here wrt AbstractBuild. What are the implications?
```

Comment:
Offhand it sounds like an esoteric feature that few Workflow users would miss. If you really need to try something similar, at least override `onStarted` to print a warning, and call `setResult(FAILURE)`; the build will continue but perhaps it will die later.